specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/networkWatcher.json

```diff
@@ -607,6 +607,249 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkWatchers/{networkWatcherName}/flowLogs/{flowLogName}": {
+      "put": {
+        "tags": [
+          "FlowLogs"
+        ],
+        "operationId": "FlowLogs_CreateOrUpdate",
+        "description": "Create or update a flow log for the specified network security group.",
```

Comment:
Do you link network security group in body? It's a bit confusing since flow log a child of network watcher, but description says nsg